app/scripts/modules/core/src/pipeline/config/stages/script/scriptExecutionDetails.html

```diff
@@ -19,7 +19,7 @@
       </div>
     </div>
 
-    <div class="row ng-scope" ng-if="stage.context.buildInfo.url">
+    <div class="row ng-scope" ng-if="!ctrl.scriptRanAndFailed && stage.context.buildInfo.url">
```

Comment:
Agreed. This is one of those things I'd like to tackle before we split everything into separate repos - refactoring pipeline configs.